github/packages_test.go

```diff
@@ -0,0 +1,676 @@
+
+func TestPackageFile_Marshal(t *testing.T) {
+	testJSONMarshal(t, &PackageFile{}, "{}")
+
+	o := &PackageFile{
+		DownloadURL: String("durl"),
+		ID:          Int64(1),
+		Name:        String("name"),
+		SHA256:      String("sha256"),
+		SHA1:        String("sha1"),
+		MD5:         String("md5"),
+		ContentType: String("ct"),
+		State:       String("state"),
+		Author: &User{
+			Login:           String("l"),
+			ID:              Int64(1),
+			URL:             String("u"),
+			AvatarURL:       String("a"),
+			GravatarID:      String("g"),
+			Name:            String("n"),
+			Company:         String("c"),
+			Blog:            String("b"),
+			Location:        String("l"),
+			Email:           String("e"),
+			Hireable:        Bool(true),
+			Bio:             String("b"),
+			TwitterUsername: String("t"),
+			PublicRepos:     Int(1),
+			Followers:       Int(1),
+			Following:       Int(1),
+			CreatedAt:       &Timestamp{referenceTime},
+			SuspendedAt:     &Timestamp{referenceTime},
+		},
+		Size:      Int64(1),
+		CreatedAt: &Timestamp{referenceTime},
+		UpdatedAt: &Timestamp{referenceTime},
+	}
+
+	want := `{
+		"download_url": "durl",
+		"id": 1,
+		"name": "name",
+		"sha256": "sha256",
+		"sha1": "sha1",
+		"md5": "md5",
+		"content_type": "ct",
+		"state": "state",
+		"author": {
+			"login": "l",
+			"id": 1,
+			"avatar_url": "a",
+			"gravatar_id": "g",
+			"name": "n",
+			"company": "c",
+			"blog": "b",
+			"location": "l",
+			"email": "e",
+			"hireable": true,
+			"bio": "b",
+			"twitter_username": "t",
+			"public_repos": 1,
+			"followers": 1,
+			"following": 1,
+			"created_at": ` + referenceTimeStr + `,
+			"suspended_at": ` + referenceTimeStr + `,
+			"url": "u"
+		},
+		"size": 1,
+		"created_at": ` + referenceTimeStr + `,
+		"updated_at": ` + referenceTimeStr + `
+	}`
+
+	testJSONMarshal(t, o, want)
+}
+
+func TestPackageRelease_Marshal(t *testing.T) {
+	testJSONMarshal(t, &PackageRelease{}, "{}")
+
+	o := &PackageRelease{
+		URL:             String("url"),
+		HTMLURL:         String("hurl"),
+		ID:              Int64(1),
+		TagName:         String("tn"),
+		TargetCommitish: String("tcs"),
+		Name:            String("name"),
+		Draft:           Bool(true),
+		Author: &User{
+			Login:           String("l"),
+			ID:              Int64(1),
+			URL:             String("u"),
+			AvatarURL:       String("a"),
+			GravatarID:      String("g"),
+			Name:            String("n"),
+			Company:         String("c"),
+			Blog:            String("b"),
+			Location:        String("l"),
+			Email:           String("e"),
+			Hireable:        Bool(true),
+			Bio:             String("b"),
+			TwitterUsername: String("t"),
+			PublicRepos:     Int(1),
+			Followers:       Int(1),
+			Following:       Int(1),
+			CreatedAt:       &Timestamp{referenceTime},
+			SuspendedAt:     &Timestamp{referenceTime},
+		},
+		Prerelease:  Bool(true),
+		CreatedAt:   &Timestamp{referenceTime},
+		PublishedAt: &Timestamp{referenceTime},
+	}
+	want := `{
```

Comment:
Please add blank line before line 140 for consistency.  Thanks.

Reply:
updated